classifiers/active_learners/random_search/code_snippet.md

```diff
@@ -33,5 +34,5 @@ class MyRandom(LearningClassifier):
     )
 
     def predict_proba(self, embeddings):
-        return self.model.predict(embeddings)
+        return self.model.predict_proba(embeddings)
```

Comment:
Why is .predict replaced with .predict_proba?

Reply:
They have different dimensionality and are quite different in the context of refinery. `predict_proba` will return a 2D array with each row representing the samople in the input data, and each column represents the predicted probability. 
`predict` returns a 1-D array, where each element represents the predicted label for a set of input data.